File: product-service/src/main/java/com/example/productservice/entities/Category.java
package com.example.productservice.entities;

import lombok.*;

import javax.persistence.*;

@Entity
@Table(name = "categories")
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class Category {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "id")
    private Long id;

    @Column(name = "name")
    private String name;

    @Column(name = "description")
    private String description;

    @Column(name = "picture_portrait_file_name")
    private String picturePortraitFileName;

    @Column(name = "picture_landscape_file_name")
    private String pictureLandscapeFileName;

    public Category(String name, String description, String picturePortraitFileName, String pictureLandscapeFileName) {
        this.name = name;
        this.description = description;
        this.picturePortraitFileName = picturePortraitFileName;
        this.pictureLandscapeFileName = pictureLandscapeFileName;
    }
}

File: web-store/src/app/features/categories/category.ts
export class Category {
  id: number;
  name: string;
  description: string;
  picturePortraitFileName: string;
  pictureLandscapeFileName: string;

  constructor(id: number, name: string, description: string, picturePortraitFileName: string, pictureLandscapeFileName: string) {
    this.id = id;
    this.name = name;
    this.description = description;
    this.picturePortraitFileName = picturePortraitFileName;
    this.pictureLandscapeFileName = pictureLandscapeFileName;
  }
}
